### DISCUSSION TITLE
Change Widgets position in ListRecords view.

### DISCUSSION THREAD

**Discussion by OsRobert** [filamentphp/filament#9412] [Help] [answered]
Hi.
Can anyone know how to move ListRecords widgets under tabs? Is this even possible?
Or create new widget where I can pass data from table. 

When I use InteractsWithPageTable in custom widget on tab change I get error:
Cannot assign null to property App\Livewire\StatsOverviewNew::$tableColumnSearches of type array

![image](https://github.com/filamentphp/filament/assets/3382187/f970869b-bc96-4f20-80fc-5aa8d2d09bf3)

**Comment by josephajibodu** [comment, marked as answer]
Widgets can only to added to the header or footer. Probably you have to create a custom view for your use-case.

But for the error, make sure you add `use ExposesTableToWidgets;` to your **List Page** and not the resources page itself.
